notebooks/228-clip-zero-shot-image-classification/228-clip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,410 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "4ad832c5-71db-456f-9702-fb8f3b489e46",
+   "metadata": {},
+   "source": [
+    "# Post-Training Quantization of OpenAI CLIP model with NNCF\n",
+    "\n",
+    "The goal of this tutorial is to demonstrate how to use the [NNCF](https://github.com/openvinotoolkit/nncf/) (Neural Network Compression Framework) 8-bit quantization in post-training mode (without the fine-tuning pipeline) to optimize a OpenVINO model for the high-speed inference via OpenVINO™ Toolkit. The optimization process contains the following steps:\n",
```

Comment:
I would slightly rephrase it and say like "The goal of this tutorial is to demonstrate how to speed up the model by applying 8-bit post-training quantization from [NNCF](https://github.com/openvinotoolkit/nncf/) (Neural Network Compression Framework)", i.e. switch the focus from NNCF to the optimization